src/main/java/uk/gov/hmcts/reform/professionalapi/controller/WelcomeController.java

```diff
@@ -28,7 +28,7 @@
 public class WelcomeController {
 
     @Value("${logging-component-name}")
-    protected static String loggingComponentName;
+    protected String loggingComponentName;
```

Comment:
use private variable everywhere if this variable is not used by child class

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/649/commits/75ebfaeeb6a31383283e6a172ce5a5834c972b07